src/components/input/input.ts

```diff
@@ -108,6 +107,93 @@ export class TextInput extends InputBase {
   /**
    * @private
    */
+  @Input() autocomplete: string;
+
+  /**
+   * @private
+   */
+  @Input() autocorrect: string;
+
+  /**
+   * @private
+   */
+  @Input() autocapitalize: string;
+
+  /**
+   * @private
+   */
+  get autofocus(): boolean { return this.autofocus; }
```

Comment:
I don't like this inlined syntax, we are trying to avoid this.
can you replace with:

```
@Input()
get autofocus(): boolean {
  return this.autofocus;
}
```